test/cookie_jar_test.js

```diff
@@ -699,4 +699,21 @@ vows
       }
     }
   })
+  .addBatch({
+    "Issue #197 - CookieJar().setCookie crashes when empty cookie is passed": {
+      "with missing parameters": {
+        topic: function() {
+          const jar = new tough.CookieJar();
+          jar.setCookie(
+            '',
+            "https://google.com",
+            this.callback
+          );
+        },
+        "results in a error being returned because of missing parameters": function(err, cookies) {
```

Comment:
We don't actually return an error, we are invoking the callback with a Null here. Should we invoke the CB with an Error()?